circle.yml

```diff
@@ -59,7 +59,7 @@ test:
         parallel: true
   post:
     - .circle/docker.sh build st2bundle
-    - .circle/docker.sh build st2actionrunner st2api st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer
+    - .circle/docker.sh build st2actionrunner st2api st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer st2garbagecollector
```

Comment:
Don't forget to include `st2garbagecollector` here: https://github.com/StackStorm/st2-packages/blob/master/circle.yml#L79

At the same time @Kami keep in mind that `docker build` and `docker push` commands are here temporarily, this logic should be moved to staging box eventually (eg. when all tests passed -> build and push Docker).